tests/rptest/services/kgo_verifier_services.py

```diff
@@ -172,7 +172,14 @@ def wait_node(self, node, timeout_sec=None):
 
         # Permit the subprocess to exit, and wait for it to do so
         self.logger.debug(f"wait_node {self.who_am_i()}: requesting shutdown")
-        self._remote(node, "shutdown")
+        try:
+            self._remote(node, "shutdown")
+        except requests.exceptions.ConnectionError:
+            # It is permitted for the remote process to abort connection and fail
+            # to send a response, as it does not wait for HTTP response to flush
+            # before shutting down.
+            pass
```

Comment:
> and wait for it to do so

any reason to uphold this, or is it implied? given the set of reasons for connection error, would it make sense to test that the condition "subprocess to exit" is true?

Reply:
After this `pass`, we fall through to the usual check for the PID to no longer exist, so we'll still block until the process has really exit.

(if for some reason it doesn't exit we'll time out waiting for the pid to disappear, but that's okay: failure to exit is unexpected and should fail the test).